features/edition-images.feature

```diff
@@ -29,7 +29,15 @@ Feature: Images tab on edit edition
     And a draft case study with images exists
     When I visit the images tab of the document with images
     And I click to hide the lead image
-    Then I should see a button to show the lead image
+    Then I should see a button to select a lead image
+
+  Scenario: User selects a new lead image
+    And a draft case study with images with the alt text "First image uploaded" and "Second image uploaded" exists
+    When I visit the images tab of the document with images
+    And I make the image with alt text "First image uploaded" the lead image
+    Then I can see that the image with alt text "First image uploaded" is the lead image
+    And I make the image with alt text "Second image uploaded" the lead image
+    Then I can see that the image with alt text "Second image uploaded" is the lead image
```

Comment:
Is there any value in covering the organisation image fallback as well? If indeed it isn't already covered

Reply:
cool have a test for that [here](https://github.com/alphagov/whitehall/pull/8445/commits/06da2ff2029df4ef74ded21e50adf0f47a1a1e4e#diff-2919dd3689ccd52b537a04bcc33a7a7936eb2502ecaaf13578b8e748dc42dfa5L29-R35)